syntax = "proto3";

message uid_generate {
    int64 uid = 1 ;
    int64 uidx = 2 ;
}